test/plugins/wasi_nn/CMakeLists.txt

```diff
@@ -70,6 +70,13 @@ foreach(BACKEND ${WASMEDGE_PLUGIN_WASI_NN_BACKEND})
         /wd4067 # unexpected tokens following preprocessor directive - expected a newline
       )
     endif()
+  elseif(BACKEND STREQUAL "neuralspeed")
+    message( STATUS "Download ML artifacts to ${CMAKE_CURRENT_BINARY_DIR}/wasinn_neural_speed_fixtures")
+    download(
+      https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF/resolve/main/llama-2-7b-chat.Q4_0.gguf
```

Comment:
Do we really need to download this huge file if we would like to do the test?
Is it possible to have a small one instead?

Reply:
I have changed. Please check, thank you.